www/build-dev-local.sh

```diff
@@ -14,8 +14,8 @@ DIR="$(dirname "$0")"
 cd "$DIR" || exit
 
 rm -rf dist/
-cp -r build dist/
+mkdir -p dist
 cp -r public/* dist/
-roc main.roc --linker=legacy -- content/ dist/
+cargo run --bin roc -- main.roc --linker=legacy -- content/ dist/
 
-simple-http-server -p 8080 --nocache --cors --index -- dist/
+npx http-server -p 8080 --nocache --cors --index -- dist/
```

Comment:
`simple-http-server` gets installed by our Nix flake, but I don't think node gets installed. We should probably either switch this back or make sure `npx` would be available without a manual install

Reply:
Avoid node at all costs is our policy :smile: 

Was there still a problem with simple-http-server @gamebox?

Reply:
I don't have it on my system - I avoid python at all costs as part of my policy

Reply:
That being said, this was not meant to be committed

Reply:
Our simple-http-server is made in rust https://github.com/TheWaWaR/simple-http-server